src/router.js

```diff
@@ -51,7 +58,17 @@ export const DootRouter = () => {
                   {currentUser && `Logged in as ${currentUser.email}`}
                 </span>
               </li>
-              <li></li>
+              <li className="nav-item">
+                <form className="form-inline">
```

Comment:
It doesn't need to be in a form. The only time I use forms is when I want to do `onSubmit`